### PR TITLE
Pause delay fix that doesnt interfere with n64 hardware

### PR DESCRIPTION
**Pull request by ariahiro64**
Reopened this and make it toggleable.

### CODE REVIEW

File: include/config/config_game.h [hunk 1]
```diff
@@ -25,4 +25,8 @@
  */
 // #define FW_SPLIT_AGE
 
+ /* Keep pause delay. Inacurate emulators have can potentially crash if enabled. */
```
Comment (Yanis42) [inline include/config/config_game.h:28]: the crash isn't from the filter but from the OoB read on the minimap right? This comment should be like ``/* Apply the anti-aliasing filter for the background of the pause menu. This adds a delay when pausing. */``

File: include/functions.h [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 #include "z64.h"
 #include "macros.h"
-
+#include "config.h"
 f32 fabsf(f32 f);
```
Comment (Yanis42) [inline include/functions.h:7]: just a small detail but I'd add some newlines after the include

File: src/code/PreRender.c [hunk 3]
```diff
@@ -768,7 +768,9 @@ void PreRender_DivotFilter(PreRender* this) {
  * Applies the Video Interface anti-aliasing filter and (optionally) the divot filter to `this->fbufSave` using
  * `this->cvgSave`
  */
-void PreRender_ApplyFilters(PreRender* this) {
+#ifdef VANILLA_PAUSE_DELAY
+
+void PreRender_Calc(PreRender* this) {
```
Comment (Yanis42) [inline src/code/PreRender.c:773]: did you renamed the function on accident?

**Review by Yanis42 — COMMENTED** [review]
Just some comments here and there

File: src/code/PreRender.c (hunk 3 above)
Comment (ariahiro64) [reply src/code/PreRender.c:773]: that was the result of rebasing the branch

**Comment by ariahiro64** [comment]
Everything should be addressed. Feel free to request more changes.

**Review by Yanis42 — APPROVED** [review]
looks good 👍

**Comment by Yanis42** [comment]
This branch has conflicts, when those are fixed I'll try this branch and merge it if there's no issues

**Review by Yanis42 — APPROVED** [review]
tried this on my N64 and got 0 issues

**Pull request merged**